primitives/runtime/src/lib.rs

```diff
@@ -535,6 +535,10 @@ pub type DispatchOutcome = Result<(), DispatchError>;
 /// - The extrinsic supplied a bad signature. This transaction won't become valid ever.
 pub type ApplyExtrinsicResult = Result<DispatchOutcome, transaction_validity::TransactionValidityError>;
 
+/// Same as `ApplyExtrinsicResult` but augmented with `PostDispatchInfo` on success.
+pub type ApplyExtrinsicResultWithPostInfo<T> =
```

Comment:
nit: this type is very verbose

Reply:
I dropped the `Post`. Cannot change it further as it would be totally inconsistent with the rest of the types then.

Reply:
:ok_hand: